### PR TITLE
Make the server compatible with Node.js >= 6

### PR DESCRIPTION
**Pull request by Anna-Livia**
[ExpressIO](https://github.com/techpines/express.io), a module that merges the capabilities of Express and SocketIO is not compatible with the latest node version.

This PR replaces the installation of ExpressIO with an express.io directory inside the project.

### CODE REVIEW

**Comment by jonathanperret** [comment]
Getting rid of express.io does sound like a good idea, thanks.

This PR seems to copy the whole of `express.io` into Paysage. This raises a couple questions:
- Do we use every feature of `express.io`? I seem to remember there was only a very small part of it that was actually useful to us.
- Did you look into the licensing issues?

**Comment by jonathanperret** [comment]
A couple more nits:
- there is a temporary file (`.app.js.swp`) lurking around, can you please remove it?
- please remove the `yarn.lock` file as well. While I am sympathetic to the general idea of moving to `yarn`, I don't think it should be part of this PR.

**Comment by Anna-Livia** [comment]
The code of the ExpressIO has been optimized to only feature elements used by paysage.
The README now give credit to the original ExpressIO project.

File: express.io/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#ExpressIO for Paysage
+
+This is a module copied from [ExpressIO](https://github.com/techpines/express.io).
+It has been modified to:
+- work with node 4;
```
Comment (jonathanperret) [inline express.io/README.md]: you mean node 6 :)

**Comment by jonathanperret** [comment]
Awesome. Let's :shipit: !

(note that the dependency to `socket.io`, previously brought by `express.io`, was missing. This was not seen in development because `testem`, one of our `devDependencies`, requires `socket.io` 🙄 )

**Pull request merged**